index.html

```diff
@@ -0,0 +1,125 @@
+          <div class="dateInfo"> <font>2022.09.13 (Tue) ~ 15 (Thu)</font> </div>
+          <div class="locate">
+            <font>@ Sports Center San Miguel de Sema Boyaca - Colombia</font>
+          </div>
+        </div>
+      </div>
+      <section class="mainProgram">
+        <h1 class="program" id="mainProgram">Main Program</h1>
+        <div class="guideBar"> </div>
+        <div class="programDetail">
+        <div class="programList">
+          <div class="programIcon"><img src="./src/images/program_icon_01.png" alt="conference"></div>
+          <div class="programTitle"><p>conference</p></div>
+          <div class="programDescription"><p>The Simple Two-step Marketing System You Can Copy</P></div>
+        </div>
+        <div class="programList">
+          <div class="programIcon"><img src="./src/images/program_icon_02.png" alt="conference"></div>
+          <div class="programTitle"><p>exhibition</p></div>
+          <div class="programDescription"><p>You will know the keys to growing professionally.</P></div>
+        </div>
+        <div class="programList">
+          <div class="programIcon"><img src="./src/images/program_icon_03.png" alt="conference"></div>
+          <div class="programTitle"><p>forum</p></div>
+          <div class="programDescription"><p>You will learn to invest in online advertising .</P></div>
+        </div>
+        <div class="programList">
+          <div class="programIcon"><img src="./src/images/program_icon_04.png" alt="conference"></div>
+          <div class="programTitle"><p>workshop</p></div>
+          <div class="programDescription"><p>Understanding The New Mechanics Of Digital Marketing</P></div>
+        </div>
+        <div class="programList">
+          <div class="programIcon"><img src="./src/images/program_icon_05.png" alt="conference"></div>
+          <div class="programTitle"><p>Party</p></div>
+          <div class="programDescription"><p>How Businesses are Using Affiliate Partnerships</P></div>
+        </div>
+      </div>
+     </section>
+      <!--  Start  dinamic Content JavaScript object -->
+
+      <section>
+          <p class="speakers_title" id="speakers">Featured Speakers</p>
+          <div class="guideBar"></div>
+      </section>
+ <section class="speakerSection" id="speaker">
+  </section>
+
+  <section class="aboutSectionPartner">
+      <div class="partners ">
+      <p class="aboutPartner">Partners</p>
+      <div class="guideBar"></div>
+      <div class="partnerIcons">
+          <img class="partnerIcon" src="./src/images/mozilla.png">
+          <img class="partnerIcon" src="./src/images/google.png">
+          <img class="partnerIcon" src="./src/images/naver.png">
+          <img class="partnerIcon" src="./src/images/airbnb.png">
+      </div>
+  </div>
+  </section>
+
+  <footer class="license">
+        <h3>Traffic & Conversion Summit</h3>
+        <p class="rigths">2017 congress of online advertising <br> Rigths Reserved</p>
+    </footer>
```

Comment:
- [ ] Please add a partners section to the homepage (kudos for removing it on the about page) and also make sure that the footer on the homepage has a white background and the footer on the about page has a dark background to match the design guidelines.
### Your design
![5](https://user-images.githubusercontent.com/94551484/190774360-2ccda162-34a8-4de8-bbfe-46065240164e.png)
### Design guideline
![f5](https://user-images.githubusercontent.com/94551484/190774794-3fd462e7-e978-4fdf-9b93-bbac1aab9a09.png)